### PR TITLE
feat(language): add malay 1k (kopipahit-1)

### PR DESCRIPTION
**Pull request by kopipahit-1**
Added malay_1k.json for Bahasa Malaysia.

### CODE REVIEW

**Comment by monkeytypegeorge** [comment]
Continuous integration check(s) failed. Please review the failing check's logs and make the necessary changes. https://github.com/monkeytypegame/monkeytype/actions/runs/6074308979

File: frontend/static/languages/malay_1k.json [hunk 1]
```diff
@@ -0,0 +1,1008 @@
+{
+  "name": "malay",
```
Comment (Miodec) [inline frontend/static/languages/malay_1k.json:2]: This needs to match filename (`malay_1k`)

File: frontend/static/languages/malay.json [hunk 2]
```diff
@@ -1,238 +1,236 @@
...
-    "di",
-    "anda",
-    "akan",
-    "apa",
-    "dia",
-    "saya",
-    "kita",
-    "untuk",
-    "mereka",
-    "ada",
```
Comment (Miodec) [inline frontend/static/languages/malay.json:22]: This pr still edits the `malay.json` file, when it should only be adding the `1k` file.

**Review by Miodec — CHANGES_REQUESTED** [review]

**Review by kopipahit-1 — COMMENTED** [review]
changes  "name": "malay", to "malay_1k".

Thank you.

**Review by kopipahit-1 — COMMENTED** [review]
Deleted malay.json file.

**Comment by monkeytypegeorge** [comment]
Continuous integration check(s) failed. Please review the failing check's logs and make the necessary changes. https://github.com/monkeytypegame/monkeytype/actions/runs/6076550419

**Comment by Miodec** [comment]
You need to revert the changes instead of deleting it.

**Review by kopipahit-1 — COMMENTED** [review]
Revert it. Thank you for the guide.

**Review by kopipahit-1 — COMMENTED** [review]
Revert it.

**Review by kopipahit-1 — COMMENTED** [review]
Revert it.

**Review by kopipahit-1 — COMMENTED** [review]
Re

**Comment by Miodec** [comment]
Thanks

**Pull request merged**